sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/DotNetSensor.java

```diff
@@ -91,6 +91,8 @@ private void executeInternal(SensorContext context) {
     List<Path> protobufPaths = reportPathCollector.protobufDirs();
     if (!protobufPaths.isEmpty()) {
```

Comment:
Can you please swap this condition to have a positive test?
```suggestion
    if (protobufPaths.isEmpty()) {
```